Development environment setup/Xdebug.md

```diff
@@ -0,0 +1,101 @@
+Xdebug is an indispensible tool when working on PHP projects. It's a very powerful debugger which can be used for code coverage when running tests as well.
+
+If you have Laravel Herd Pro installed, Xdebug is already installed and configured for you. If you don't have Laravel Herd Pro, you can install Xdebug manually.
```

Comment:
Is it really installed with Herd, I'm not quite sure about that and Herd docs do not specify that directly anywhere AFAIK

Reply:
It is 
<img width="848" alt="image" src="https://github.com/infinum/wordpress-handbook/assets/8638515/ca0a76b1-f1d8-4051-9ec2-69b00c762da6">

Reply:
Oh 🤦🏼 Thanks
It is written in the FAQ 😶